.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+
+on: [push, pull_request]
+name: CI
+jobs:
+  test:
+    name: Test
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: test
+        run: |
+          go install golang.org/x/lint/golint@latest &&
```

Comment:
Since golint is deprecated, this should be `go get -u honnef.co/go/tools/cmd/staticcheck@latest`

Reply:
Or `go install honnef.co/go/tools/cmd/staticcheck@latest`

Reply:
Doesn't seem like this change got applied for some reason 🤷